include/z64cutscene_commands.h

```diff
@@ -11,445 +11,266 @@
- *   s16 startFrame (s), s16 endFrame (e)
- * FORMAT
- *   Capital U is Unused , may be consistently zero
- *   00000006 0001ssss eeee0000
- *   size = 0xC
- */
-#define CS_CAM_FOCUS_POINT_PLAYER_LIST CS_CAM_AT_REL_TO_PLAYER_LIST
-#define CS_CAM_AT_REL_TO_PLAYER_LIST(startFrame, endFrame) \
-    CS_CMD_CAM_AT_REL_TO_PLAYER, CMD_HH(0x0001, startFrame), CMD_HH(endFrame, 0x0000)
-/**
- * ARGS
- *   s8 continueFlag (c), s8 roll (r), s16 frame (f), f32 viewAngle (a),
- *   s16 xPos (x), s16 yPos (y), s16 zPos (z)
- * FORMAT
- *   Capital U is Unused
- *   ccrrffff aaaaaaaa xxxxyyyy zzzzUUUU
- *   size = 0x10
+ * Declares a list of `CS_MISC` entries.
  */
-#define CS_CAM_FOCUS_POINT_PLAYER CS_CAM_AT_REL_TO_PLAYER
-#define CS_CAM_AT_REL_TO_PLAYER(continueFlag, roll, frame, viewAngle, xPos, yPos, zPos, unused) \
-    CMD_BBH(continueFlag, roll, frame), CMD_F(viewAngle), CMD_HH(xPos, yPos), CMD_HH(zPos, unused)
+#define CS_MISC_LIST(entries) \
+    CS_CMD_MISC, CMD_W(entries)
 
 /**
- * ARGS
- *   s16 unk (u), s16 startFrame (s), s16 endFrame (e)
- * FORMAT
- *   Capital U is Unused
- *   00000007 uuuussss eeeeUUUU
- *   size = 0xC
+ * Various miscellanious commands.
+ * @see `CutsceneMiscCommand` enum for the different types of commands.
  */
-#define CS_CMD_07_LIST(unk, startFrame, endFrame, unused) \
-    CS_CMD_07, CMD_HH(unk, startFrame), CMD_HH(endFrame, unused)
-
-/**
- * ARGS
- *   s8 continueFlag (c), s8 roll (r), s16 frame (f), f32 viewAngle (a),
- *   s16 xPos (x), s16 yPos (y), s16 zPos (z)
- * FORMAT
- *   Capital U is Unused
- *   ccrrffff aaaaaaaa xxxxyyyy zzzzUUUU
- *   size = 0x10
- */
-#define CS_CMD_07(continueFlag, roll, frame, viewAngle, xPos, yPos, zPos, unused) \
-    CMD_BBH(continueFlag, roll, frame), CMD_F(viewAngle), CMD_HH(xPos, yPos), CMD_HH(zPos, unused)
+#define CS_MISC(cmd, startFrame, endFrame, unused0, unused1, unused2, unused3, unused4, unused5, unused6, unused7, unused8, unused9, unused10) \
+    CMD_HH(cmd, startFrame), CMD_HH(endFrame, unused0), \
+    CMD_W(unused1), CMD_W(unused2), CMD_W(unused3), CMD_W(unused4), CMD_W(unused5), \
+    CMD_W(unused6), CMD_W(unused7), CMD_W(unused8), CMD_W(unused9), CMD_W(unused10)
 
 /**
- * ARGS
- *   s16 unk (u), s16 startFrame (s), s16 endFrame (e)
- * FORMAT
- *   Capital U is Unused
- *   00000008 uuuussss eeeeUUUU
- *   size = 0xC
+ * Declares a list of `CS_LIGHT_SETTING` entries.
  */
-#define CS_CMD_08_LIST(unk, startFrame, endFrame, unused) \
-    CS_CMD_08, CMD_HH(unk, startFrame), CMD_HH(endFrame, unused)
+#define CS_LIGHT_SETTING_LIST(entries) \
+    CS_CMD_LIGHT_SETTING, CMD_W(entries)
 
 /**
- * ARGS
- *   s8 continueFlag (c), s8 roll (r), s16 frame (f), f32 viewAngle (a),
- *   s16 xPos (x), s16 yPos (y), s16 zPos (z)
- * FORMAT
- *   Capital U is Unused
- *   ccrrffff aaaaaaaa xxxxyyyy zzzzUUUU
- *   size = 0x10
+ * Changes the lights to the specified setting.
+ * @note This will only work for `LIGHT_MODE_SETTINGS`, it will not override time-based light configs.
```

Comment:
```suggestion
 * @note This will only work for `LIGHT_MODE_SETTINGS`, it will not override time-based light settings.
```
... I think (i.e. "config-and-time-based light settings")

Reply:
the config is what matters for time-based scenarios. They do eventually map to settings under the hood, but the point is that you cant change `envCtx->lightConfig` with this

Reply:
I meant that as you know the settings are time-based, not the configs
Both time and config go into picking a setting in `light_mode_time`

I guess if you meant "cant change `lightConfig` with this" then
```suggestion
 * @note This will only work for `LIGHT_MODE_SETTINGS`, it will not override light configs.
```
or
```suggestion
 * @note This will only work for `LIGHT_MODE_SETTINGS`, it will not override light configs in the case of `LIGHT_MODE_TIME`.
```